Gems/AWSGameLift/Code/AWSGameLiftClient/Source/AWSGameLiftClientManager.cpp

```diff
@@ -86,16 +82,7 @@ namespace AWSGameLift
         }
 
         // Set up client endpoint or region
-        AZStd::string localEndpoint = "";
```

Comment:
Heads up, compile errors due to code still referencing localEndpoint. (see line #104)

Reply:
Thanks, fixed!